spec/support/shared_examples/models/shared_examples_for_clar.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.shared_context 'when set up for #unused_materials_applicable? tests' do |options|
```

Comment:
The shared examples here:

-  seem confused and are confusing. perhaps just simplfy and have repeated, not shared examples which can be more easily followed.
- the name "a claim [not] eligible for unused materials fee" is confusing too since it clashes with the notion of eligible misc fees - i.e. other claim types are eligible for unused materials. These examples are purely related to eligibility for the unused material **signposting**